device/arista/x86_64-arista_7060x6_64pe_b/Arista-7060X6-64PE-B-P32O64/sai.profile

```diff
@@ -1,3 +1,4 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th5-a7060x6-64pe.config.bcm
 SAI_NUM_ECMP_MEMBERS=96
 SAI_NHG_HIERARCHICAL_NEXTHOP=false
+SAI_SWITCH_SWITCHING_MODE_CUT_THROUGH=true
```

Comment:
why cut_through in sai.profile and smart SF in bcm file for some SKUs?

Reply:
@ccroy-arista could you check and confirm sai profile and bcm mapping which one has precedence? If sai.profile changes are not required to enable SmartSAF, could you remove them?